__bass.py

```diff
@@ -24,11 +24,13 @@ def gen_script():
                        .check_output(['bash', '-c', command])
                        .split(divider, 1))
     new_env = new_env.lstrip().splitlines()
-
+    
+    new_env = [line for line in new_env if '{' not in line and '}' not in line]
```

Comment:
No, it is still needed. Basically, at least the } can appear alone on a line by itself. You can try installing the mc package to see a case where it is required. (Though it may depend on your distro. The OpenSuse package does have the problematic pattern. I don't know about others.)